UI/TechTreeWnd.cpp

```diff
@@ -564,6 +565,9 @@ class TechTreeWnd::LayoutPanel : public GG::Wnd {
     void TreeZoomedSlot(int move);
     void TreeZoomInClicked();
```

Comment:
having two sets of these functions seems unnecessary, but probably OK to avoid awkward uses of boost::bind to discard return values when not needed...